dogfood/README.md

```diff
@@ -1,4 +1,4 @@
-# Installing protoc
+ # Installing protoc
```

Comment:
why?

Reply:
Must have been a mistake. I'll change it back.